spec/helpers/quadicon_helper_spec.rb

```diff
@@ -15,18 +15,42 @@
   end
 
   describe "#render_quadicon_text" do
+    before(:each) do
+      @settings = {:display => {:quad_truncate => "m"}}
+    end
+
     subject { helper.render_quadicon_text(item, row) }
 
     let(:item) do
       FactoryGirl.create(:vm_vmware)
     end
 
     let(:row) do
-      Ruport::Data::Record.new(:id => 10000000000534)
+      Ruport::Data::Record.new(:id => 10_000_000_000_534, "name" => name)
```

Comment:
I would not fix rubocops on a backport...the backport should be as close to the original PR as possible to keep everything consistent.